service/src/main/java/uk/gov/hmcts/reform/fpl/validation/interfaces/time/DateNotInFuture.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.fpl.validation.interfaces.time;
+
+import uk.gov.hmcts.reform.fpl.validation.validators.time.DateNotInFutureValidator;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = DateNotInFutureValidator.class)
+public @interface DateNotInFuture {
```

Comment:
maybe for consistency it can be called PastOrPresentDate

Reply:
👍